Code/cmake/Modules/RDKitVersion.cmake

```diff
@@ -0,0 +1,21 @@
+set(RDKit_Year "2018")
```

Comment:
Changing these values in this file seems odd (and a bit opaque) to me.
Will this patch work if we leave the definitions of these values in the root `CMakeLists.txt` file?

Reply:
Yes,I think it should still work. I’ll check it later.